File: College-Sports-WebApp/ClientApp/src/api/models/airing.ts
/* tslint:disable */
/* eslint-disable */
import { Image } from '../models/image';
import { Policy } from '../models/policy';
import { ProductLinks } from '../models/product-links';
import { ProgramCategory } from '../models/program-category';
import { Properties } from '../models/properties';
export interface Airing {
  appAiringLink?: string | null;
  appGameLink?: string | null;
  artworkUrl?: string | null;
  createdBy?: string | null;
  createdOn?: string;
  end?: string;
  externalId?: string | null;
  id?: number;
  images?: Array<Image> | null;
  lastModifiedBy?: string | null;
  lastModifiedOn?: string;
  network?: string | null;
  networkId?: string | null;
  network_displayName?: string | null;
  network_shortName?: string | null;
  playableUrl?: string | null;
  policy?: Policy;
  policyUrl?: string | null;
  productLinks?: ProductLinks;
  program?: string | null;
  program_categories?: Array<ProgramCategory> | null;
  program_eventId?: string | null;
  program_eventUrl?: string | null;
  program_firstPresented?: string;
  program_language?: string | null;
  program_originalAirDate?: string;
  program_shortTitle?: string | null;
  properties?: Properties;
  start?: string;
  webAiringLink?: string | null;
  webGameLink?: string | null;
  withinPlayWindow?: boolean;
}

File: College-Sports-WebApp/Database/Models/Airing.cs
using System.Collections.Generic; 
using System; 
namespace College_Sports_WebApp.Database.Models{ 

    public class Airing
    {
        public string createdBy { get; set; }
        public DateTime createdOn { get; set; }
        public string lastModifiedBy { get; set; }
        public DateTime lastModifiedOn { get; set; }
        public string artworkUrl { get; set; }
        public string externalId { get; set; }
        public Properties properties { get; set; }
        public ProductLinks productLinks { get; set; }
        public string id { get; set; }
        public string program { get; set; }
        public string playableUrl { get; set; }
        public string policyUrl { get; set; }
        public DateTime start { get; set; }
        public DateTime end { get; set; }
        public string network { get; set; }
        public string networkId { get; set; }
        public Policy policy { get; set; }
        public bool withinPlayWindow { get; set; }
        public List<Image> images { get; set; }
        public string network_shortName { get; set; }
        public string network_displayName { get; set; }
        public string program_language { get; set; }
        public string program_eventId { get; set; }
        public string program_eventUrl { get; set; }
        public string program_shortTitle { get; set; }
        public DateTime program_originalAirDate { get; set; }
        public DateTime program_firstPresented { get; set; }
        public string webAiringLink { get; set; }
        public string appAiringLink { get; set; }
        public List<ProgramCategory> program_categories { get; set; }
        public string webGameLink { get; set; }
        public string appGameLink { get; set; }
    }

}